Sources/DocCDocumentation/DocCDocumentation.docc/customizing-your-documentation-appearance.md

```diff
@@ -0,0 +1,244 @@
+# Customizing Your Documentation Appearance
+
+Customize the look and feel of your documentation webpages.
+
+## Overview
+
+By default, rendered documentation webpages produced by DocC come with a default
+visual styling. If you wish, you may make adjustments to this styling by adding
+an optional `theme-settings.json` file to the root of your documentation catalog
+with some configuration. This file is used to customize various things like
+colors and fonts.  You can even make changes to the way that specific elements
+appear, like buttons, code listings, and asides. Additionally, some metadata for
+the website can be configured in this file, and it can also be used to opt in or
+opt out of certain default rendering behavior.
+
+If you're curious about the full capabilities of the `theme-settings.json` file,
+you can check out the latest Open API specification for the it [here][1], which
+documents all valid settings that could be present.
+
+Let's walk through some basic examples of customizing the appearance of
+documentation now.
+
+### Colors
+
+DocC-Render utilizes [CSS custom properties][2] (CSS variables) to define the
```

Comment:
This documentation catalog doesn't refer to "DocC Render", which might be overly specific. If we use the term, we should explain it, but otherwise, should we prefer "DocC" or "DocC websites"?

Reply:
Good point. I might need to use it, but I agree that I should explain it if so. It would be preferable to not mention it entirely, but I'm not sure we could get away with that. I'll look into it.

Reply:
I just went with DocC on its own: b957dc74cdda1a7f91582a57c83f81c8c6655f0b